### ISSUE TITLE
False negatives in local links

### ISSUE THREAD

**Issue by KhudaDad414** [asyncapi/.github#158]
#### Describe the bug
Recently introduced check for Links that run weekly gives false negatives in all of the local markdown links. 
as you can see in the latest run (https://github.com/asyncapi/generator/runs/6050625807?check_suite_focus=true) all of the local links that point to some section of the same file or other local files, gives `404` error. 

#### Expected behaviour
They should pass since I can click on them and they work perfectly.

#### Root of the issue
After checking the links with other link checkers online, I see all of them give 404 errors unless the link is opened by a browser. I assume it is some kind of protection against bots by GitHub.

#### Possible Solution
I guess we should ignore local links?

**Comment by derberg** [comment]
It is a problem with underlying links validator, have a look at https://github.com/gaurav-nelson/github-action-markdown-link-check/issues/127 and fix is not close to be merged https://github.com/gaurav-nelson/github-action-markdown-link-check/pull/129. We might need to switch back to original action from a fork I created recently as they already fixed the issue https://github.com/gaurav-nelson/github-action-markdown-link-check/issues/132 and maybe we will resolve the problem.

Why I think switching back to original action will work? because to fix the issue, when I created a fork, I fixed things on master, and therefore I was using code base of `1.0.14` that has issues with anchors
